client/src/components/payment/payment-complete-modal.vue

```diff
@@ -82,11 +82,6 @@ export default class PaymentCompleteModal extends Mixins(NameRequestMixin, Payme
         // Then complete the payment
         await this.completePayment(nrId, sessionPaymentId, sessionPaymentAction)
       } else {
-        errorModule.setAppError({
```

Comment:
Something to keep in mind here: it may be possible to look at `sbcPaymentStatus` and `paymentStatus` (as on line 81 above) and determine that, yes, the user returned from PayBC but their payment was cancelled or failed. And in this case, we could give the user a useful dialog.

Reply:
Definitely, i agree 100%.
Question, should we do this now or after Payment updates are in?